### ISSUE TITLE
search_opensearch.php wrongly indicates that each suggestion has exactly 1 result

### ISSUE THREAD

**Issue by Krinkle** [doxygen/doxygen#10017]
**Describe the bug** (Actual behaviour):

<https://doc.wikimedia.org/mediawiki-core/master/php/search_opensearch.php?v=json>
> `["foo", ["footer", "footers", "footerhtml", …, "fooey"],["1 result", "1 result", "1 result", … "1 result"]]`

There is also a warning in the server-side PHP error.log:
```
PHP Warning:  array_key_exists(): The first argument should be either a string or an integer)
```

**Expected behavior**

> `["foo", ["footer", "footers", "footerhtml", …, "fooey"],["33 results", "4 results", "5 results", … "1 result"]]`

**Version**
1.9.6

**Comment by albert-github** [comment]
Please don't add a link to a pull request that closes the issue.
When merged doxygen will close the issue at the next release after the merge.

**Comment by Krinkle** [comment]
@albert-github Ack, I've updated the commit message to say "Ref" instead of "Fixes" to avoid automatic closure.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.7.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
